webui/src/app/components/appdetail/appdetail.component.html

```diff
@@ -50,7 +50,7 @@ <h5 class="oui-header_5">
         <hr>
 
         <h5 class="oui-header_5">
-          <i class="oui-icon oui-icon-user_line" aria-hidden="true"></i>
+          <app-help-widget key="help-what-is-a-contributor"></app-help-widget><i class="oui-icon oui-icon-user_line" aria-hidden="true"></i>
```

Comment:
The help icon should appear after the "Authors" string.

Reply:
Done